### ISSUE TITLE
Make it possible to hide dock-icon

### ISSUE THREAD

**Issue by hkjels** [keybase/client#3907]
I see that you guys have your hands full. Please feel free to ignore this "issue" for some time 😀

my log id: 2451999ac1d81b90ecab901c

**Comment by gabriel** [comment]
If you do File > Quit while the app has focus, the dock icon should go away.
![2016-08-17 at 8 07 pm](https://cloud.githubusercontent.com/assets/2669/17760670/4f6023c0-64b6-11e6-93bf-a67b72af1065.png)

If not there is a bug.

You can try using a prerelease build:
https://prerelease.keybase.io/darwin/Keybase-1.0.17-20160818001424%2B1850da5.dmg

**Comment by hkjels** [comment]
So, using the dropdown you mentioned works just fine. However, using "⌘q" will quit Keybase entirely. Feel free to close the issue if that's the behavior your aiming for.

**Comment by jtokoph** [comment]
I came across this issue when trying to figure out how to hide the window and dock icon. I figured I would voice my opinion on the issue. I would argue that having CMD + Q behave differently than Keybase -> Quit is confusing for mac users. Those two actions have always been identical, especially since the menu option shows cmd+Q as the keyboard shortcut.

<img width="168" alt="screen shot 2017-05-25 at 2 00 47 pm" src="https://cloud.githubusercontent.com/assets/346225/26470451/93f51fe4-4152-11e7-931e-5eb4683369a5.png">

One option is to have cmd+q and the quit menu behave the same way, but have cmd+w and closing the window both close the window and remove the dock icon.
